lab-matt/lib/first.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Great job!  I really like how you pass null back into the callback when there is an error.